Resources/Prototypes/_LostParadise/Datasets/nightmare.yml

```
@@ -0,0 +1,305 @@
+  - МЕНЯ НАКРЫЛО?
+  - на поводке
+  - кишки
+  - кишки, кругом одни кишки!
+  - лужа крови
+  - металлический привкус
+  - неописуемый ужас
+  - сердце лопается!
+  - тиканье
+  - безжизненный взгляд
+  - где я?
+  - пустой коридор 
```

yamllint

[error] trailing spaces

(trailing-spaces)